provision-base.sh

```diff
@@ -253,6 +261,26 @@ EO_CSH_SHELL
 	grep -q PS1 /etc/csh.cshrc || echo "$_cconf" | tee -a /etc/csh.cshrc
 }
 
+config_zsh_shell()
+{
+	tell_status "makeing zsh more comfy with ZIM"
+
+    #fetch -o "$BASE_MNT/root/zim-master.zip" https://github.com/Eriner/zim/archive/master.zip
+    fetch -o "$BASE_MNT/root/zim.tar.gz" https://github.com/Infern1/Mail-Toaster-6/raw/zsh_shell/contrib/zim.tar.gz
+
+    cd "$BASE_MNT/root" || exit
+    tar -xzf zim.tar.gz
+    rm -rf .zim .zimrc .zlogin .zshrc
+    mv -f zim .zim/
+    stage_exec cp /root/.zim/templates/zimrc /root/.zimrc
+    stage_exec cp /root/.zim/templates/zlogin /root/.zlogin
+    stage_exec cp /root/.zim/templates/zshrc /root/.zshrc
+    stage_exec zsh -c '. /root/.zshrc;  source /root/.zlogin'
+    sed -i .bak  's/zprompt_theme='\''steeef'\''/zprompt_theme='\''liquidprompt'\''/' "$BASE_MNT/root/.zimrc"
+
```

Comment:
Rather than configuring zsh for the root user, maybe configure it for the jail by having zim.tar.gz contain system wide config zsh config files dropped into /etc with `tar -C /etc ...`?